test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -428,6 +496,24 @@ private V1Deployment getPravegaOperatorDeployment() {
                                         .build();
     }
 
+    public CompletableFuture<V1ConfigMap> setupTLS() {
+            V1ConfigMap configMap = Futures.getThrowingException(k8sClient.getConfigMap(PRAVEGA_CONTROLLER_CONFIG_MAP, NAMESPACE));
+            return k8sClient.deleteConfigMap(PRAVEGA_CONTROLLER_CONFIG_MAP, NAMESPACE)
+                        .thenCompose(v -> k8sClient.createConfigMap(NAMESPACE, addDefaultTlsConfiguration(configMap)));
+    }
+
+    private V1ConfigMap addDefaultTlsConfiguration(V1ConfigMap configMap) {
+            return configMap
+                        .putDataItem("TLS_ENABLED", "true")
+                        .putDataItem("TLS_KEY_FILE", "/opt/pravega/conf/server-key.key")
+                        .putDataItem("TLS_CERT_FILE", "/opt/pravega/conf/server-cert.crt")
+                        .putDataItem("TLS_TRUST_STORE", "/opt/pravega/conf/server-cert.crt")
```

Comment:
If the server cert is signed by a CA, using the CA's certificate as the truststore is better as that'd reflect real-world scenarios better.